src/client.rs

```diff
@@ -208,6 +212,17 @@ impl Client {
             last_ptn: -1,
         }
     }
+
+    /// Build an admin client.
+    pub fn admin_client(&self) -> Admin {
```

Comment:
Maybe just `admin`? EG, `client.admin()` to access the admin interface. Thoughts?